assets/src/edit-story/components/panels/test/textStyle.js

```diff
@@ -274,8 +290,8 @@ describe('Panels/TextStyle', () => {
 
     it('should update multi padding without lock and different padding', () => {
       const { getByTestId, pushUpdateForObject } = renderTextStyle([
-        textElement,
-        textDifferentPadding,
+        unlockPaddingTextElement,
+        unlockPaddingTextDifferentPadding,
       ]);
       fireEvent.click(getByTestId('padding.lock'));
```

Comment:
Does this still make sense? Does this actually click the lock and then shouldn't the test fail, because changing the lock at this point would toggle it to true, as it's inited with false?

All these tests (for both panels) really should be updated properly to correctly assert, that this new feature is:

* Correctly persisted to store
* Correctly used in calculations

Reply:
I think we don't need `pushUpdate` function test for this panel test like other cases. `pushUpdate` function test already added on `designPanel` so I added test cases for lock ratio button called with correct value.